types/alexa-voice-service/alexa-voice-service-tests.ts

```diff
@@ -0,0 +1,34 @@
+import * as AVS from "alexa-voice-service";
+
+const options = {
+    debug: true,
+    clientId: "",
+    clientSecret: "",
+    deviceId: "",
+    refreshToken: "",
+};
+
+const avsInstance = new AVS(options);
```

Comment:
I think `AVS` doesn't have default export so you should use `new AVS.AVS(options);`